website/server/models/chat.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
+
+const schema = new mongoose.Schema({
+  timestamp: Date,
+  user: String,
+  text: String,
+  contributor: {type: mongoose.Schema.Types.Mixed},
+  uuid: String,
+  id: String,
+  groupId: {type: String, ref: 'Group'},
+});
+
```

Comment:
We should add the BasePlugin here before merging, like the other models

Reply:
Will do!